app/controllers/practice_resources_controller.rb

```diff
@@ -0,0 +1,41 @@
+class PracticeResourcesController < ApplicationController
+  before_action :authenticate_user!, except: [:download]
+  before_action :set_practice_resource, only: [:download]
+
+  def download
+    if @practice_resource
+      redirect_to @practice_resource.attachment_s3_presigned_url
+    else
+      render plain: "Resource not found", status: :not_found
+    end
+  end
+
+  private
+
+  def set_practice_resource
+    practice = Practice.find_by(id: resource_params[:practice_id])
+
```

Comment:
@camillevilla do we want some kind of logic here that prevents unpublished innovations from having their download links made available? If unpublished I would think we would only want to allow site admins or innovation editors to access. Same with public facing vs not, do we want to authenticate for non-public facing innovations?